stimela/cargo/cab/politsiyakat/parameters.json

```diff
@@ -0,0 +1,118 @@
+{
+    "task": "politsiyakat", 
+    "base": "stimela/politsiyakat", 
+    "tag": "0.3.1", 
+    "description": "Routines for automated post-1GC error detection and mitigation", 
+    "prefix": "--", 
+    "binary": "python -m politsiyakat", 
+    "msdir": "true", 
+    "parameters": [
+        {
+            "info": "Specify which suite to search for the required task", 
+            "dtype": "str", 
+            "default": "antenna_mod", 
+            "name": "tasksuite", 
+            "choices": [
+                "antenna_mod"
+            ]
+        }, 
+        {
+            "info": "name of task to execute", 
+            "name": "command", 
+            "default": null, 
+            "dtype": "str", 
+            "required": "true", 
+            "mapping": "task", 
+            "choises": [
+                "flag_antennas_with_few_baselines_remaining", 
+                "flag_excessive_delay_error", 
+                "flag_excessive_amp_phase_error", 
+                "read_ms_maintable_chunk", 
+                "check_ms"
+            ]
+        }, 
+        {
+            "info": "Chunk id to read", 
+            "dtype": "int", 
+            "default": null, 
+            "name": "chunk_id"
+        }, 
+        {
+            "info": "Minimum unflagged baselines per antenna before clipping antenna", 
+            "dtype": "int", 
+            "default": null, 
+            "name": "min_bls_per_ant"
+        }, 
+        {
+            "info": "Measurement set", 
+            "dtype": "file", 
+            "default": null, 
+            "name": "msname", 
+            "io": "msfile"
+        }, 
+        {
+            "info": "Data column in MS", 
+            "dtype": "str", 
+            "default": "CORRECTED_DATA", 
+            "name": "data_column", 
+            "mapping": "data_column"
+        }, 
+        {
+            "info": "Field(s)", 
+            "dtype": "str", 
+            "default": null, 
+            "name": "field", 
+            "mapping": "field"
+        }, 
+        {
+            "info": "Calibrator Field(s), subset of 'field'", 
+            "dtype": "str", 
+            "default": null, 
+            "name": "cal_field", 
+            "mapping": "cal_field"
+        }, 
+        {
+            "info": "Valid phase range", 
+            "dtype": "str", 
+            "default": "-15.0~15.0", 
+            "name": "phase_range_clip"
+        }, 
+        {
+            "info": "Maximum fractional offset from median power in time and field before clipping", 
+            "dtype": "float", 
+            "default": "0.4", 
+            "name": "amp_frac_clip"
+        }, 
+        {
+            "info": "Maximum fraction of data flagged due to amplutude or phase accross calibrators before clipping baseline", 
```

Comment:
s/amplutude/amplitude

Reply:
Remove this cab. It is obsolete and replaced with other tasks.